theatre/studio/src/panels/DetailPanel/DetailPanel.tsx

```diff
@@ -22,7 +22,8 @@ const Container = styled.div<{pin: boolean}>`
   position: fixed;
   right: 8px;
   top: 50px;
-  width: 236px;
+  // Temporary, see comment about CSS grid in SingleRowPropEditor.
+  width: 280px;
```

Comment:
Yeah, it's not ideal that we are specifying widths statically because it leads to clipping. I don't imagine we'll fix this soon, but I still wanted to mark it as such.